docs/pages/components/calendar.md

```diff
@@ -18,747 +18,784 @@ Classes available for `.fd-calendar__item`. All are available to dates, months,
-				</td>
-				<td class="fd-calendar__item" role="gridcell">
-					<span class="fd-calendar__text" role="button">25</span>
-				</td>
-			</tr>
-			<tr class="fd-calendar__row">
-				<td class="fd-calendar__item is-blocked" role="gridcell">
-					<span class="fd-calendar__text" role="button">26</span>
-				</td>
-				<td class="fd-calendar__item is-blocked" role="gridcell">
-					<span class="fd-calendar__text" role="button">27</span>
-				</td>
-				<td class="fd-calendar__item is-blocked" role="gridcell">
-					<span class="fd-calendar__text" role="button">28</span>
-				</td>
-				<td class="fd-calendar__item" role="gridcell">
-					<span class="fd-calendar__text" role="button">29</span>
-				</td>
-				<td class="fd-calendar__item" role="gridcell">
-					<span class="fd-calendar__text" role="button">30</span>
-				</td>
-				<td class="fd-calendar__item fd-calendar__item--other-month" role="gridcell">
-					<span class="fd-calendar__text" role="button">1</span>
-				</td>
-				<td class="fd-calendar__item fd-calendar__item--other-month" role="gridcell">
-					<span class="fd-calendar__text" role="button">2</span>
-				</td>
-			</tr>
-			</tbody>
-			</table>
-		</div>
-	</div>
+    <div class="fd-calendar__content fd-calendar__content--dates" id="IO0cp341">
+        <table class="fd-calendar__table" role="grid">
+            <thead class="fd-calendar__group">
+            <tr class="fd-calendar__row">
+                <th class="fd-calendar__item fd-calendar__item--side-helper"></th>
+                <th class="fd-calendar__item fd-calendar__item--side-helper">
+                    <span class="fd-calendar__text" role="button">S</span>
+                </th>
+                <th class="fd-calendar__item fd-calendar__item--side-helper">
+                    <span class="fd-calendar__text" role="button">M</span>
+                </th>
+                <th class="fd-calendar__item fd-calendar__item--side-helper">
+                    <span class="fd-calendar__text" role="button">T</span>
+                </th>
+                <th class="fd-calendar__item fd-calendar__item--side-helper">
+                    <span class="fd-calendar__text" role="button">W</span>
+                </th>
+                <th class="fd-calendar__item fd-calendar__item--side-helper">
+                    <span class="fd-calendar__text" role="button">T</span>
+                </th>
+                <th class="fd-calendar__item fd-calendar__item--side-helper">
+                    <span class="fd-calendar__text" role="button">F</span>
+                </th>
+                <th class="fd-calendar__item fd-calendar__item--side-helper">
+                    <span class="fd-calendar__text" role="button">S</span>
+                </th>
+            </tr>
+            </thead>
+            <tbody class="fd-calendar__group">
+            <tr class="fd-calendar__row">
+                <td class="fd-calendar__item fd-calendar__item--side-helper" role="gridcell">
```

Comment:
The `gridcell` role should be applied to the focusable elements in the grid (the dates). It probably doesn't hurt to leave it on the `--side-helper`s.

The rest of the markup appears consistent with the previous a11y changes.